torch/csrc/jit/passes/quantization/finalize.cpp

```diff
@@ -7,6 +7,12 @@
 namespace torch {
 namespace jit {
 
+struct PatternReplaceInfo {
```

Comment:
we have this in quantization_patterns.h

Reply:
might make sense to move the patterns there as well

Reply:
I can do this in a follow-up PR and move all patterns in this file there.